plugins/tray.c

```diff
@@ -631,7 +631,7 @@ static GtkWidget *tray_constructor(LXPanel *panel, config_setting_t *settings)
     /* Add GDK event filter. */
     gdk_window_add_filter(NULL, (GdkFilterFunc) tray_event_filter, tr);
     /* Reference the window since it is never added to a container. */
-    tr->invisible = g_object_ref_sink(G_OBJECT(invisible));
+    tr->invisible = (GtkWidget *) g_object_ref_sink(G_OBJECT(invisible));
```

Comment:
I'm not sure, so forgive me if I'm wrong, but should it be GTK_WIDGET(...) instead to be more idiomatic?

Reply:
Thanks! I tested the build with GTK_WIDGET(...) and I have updated the PR.